pyproject.toml

```diff
@@ -120,6 +122,7 @@ all_extras_pandas2 = [
   "dtw-python",
   'esig==0.9.7; python_version < "3.10"',
   'filterpy>=1.4.5; python_version < "3.11"',
+  'FrEIA; python_version < "3.12"',
```

Comment:
I think python package names ignore capitalization

Reply:
I took the naming from pypi: https://pypi.org/project/FrEIA/